tests/test_statcast_pitching_spin.py

```diff
@@ -0,0 +1,126 @@
+import pybaseball.statcast_pitcher_spin as spin
+import unittest
+import pandas as pd
+import numpy as np
+import logging
+
+SIG_DIG = 4 # Required because of read_csv bug in pandas
+
+rounding_error_columns = ['vxR', 'vyR', 'vxbar', 'vybar', 'vbar']
+
+class TestStatcastPitchingSpinCalcs(unittest.TestCase):
+
+	# Helper Methods
+	def compare_columns(self, df1, df2, name):
+		return df1[name].equals(df2[name])
```

Comment:
Probably safer to use `assert_series_equal` here (though I do not have a lot of experience with it):

https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.testing.assert_series_equal.html

Reply:
Also if this is done that way, I would just eliminate this function and do the `assert_series_equal` directly in your tests.